Gems/Atom/Feature/Common/Registry/splash_screen.setreg

```diff
@@ -0,0 +1,16 @@
+{
+    "O3DE": {
```

Comment:
I think we should consider adding this `.setreg` in the project templates e.g. https://github.com/o3de/o3de/tree/development/Templates/DefaultProject/Template/Registry